test/runtests.jl

```diff
@@ -1,29 +1,41 @@
 using TensorOperations
 using LinearAlgebra
 using Test
+using TestExtras
 using Random
 using Aqua
 
 Random.seed!(1234567)
-
-include("tensoropt.jl")
-include("auxiliary.jl")
-include("macro_kwargs.jl")
-include("methods.jl")
-include("tensor.jl")
-include("ad.jl")
+@timedtestset "tensoropt" verbose = true begin
```

Comment:
is there a reason to switch to `@timedtestset` here? I think TensorOperations is now julia 1.8+, so this really should not make any difference

Reply:
No, I used my local TestExtras branch which included your `@include` macro, but then I changed the test structure again to not need this. I am happy to get rid of this again.